src/main/java/seedu/address/commons/core/range/Range.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.commons.core.range;
+
+import seedu.address.commons.core.index.Index;
+
+public class Range {
+    private Index start;
+    private Index end;
+
+    /**
+     * Constructs a range from start to end, inclusive.
```

Comment:
Should we change to "Constructs an inclusive range from start index to end index"?